csaf_2.1/prose/edit/src/tests-03-informative.md

```diff
@@ -508,117 +508,4 @@ The relevant path for this test is:
 
-### Usage of Private SSVC Decision Point Namespace in non TLP:CLEAR Document
-
-For each SSVC decision point given under `selections`, it MUST be tested the `namespace` is not a private one if the document is not labeled `TLP:CLEAR`.
-
-The relevant path for this test is:
-
-```
-   /vulnerabilities[]/metrics[]/content/ssvc_v1/selections[]/namespace
-```
-
-*Example 1 (which fails the test):*
-
-```
-  {
-    "document": {
-      // ...
-      "distribution": {
-        "tlp": {
-          "label": "GREEN"
-        }
-      },
-      // ...
-    }
-    "vulnerabilities": [
-      {
-        "cve": "CVE-1900-0001",
-        "metrics": [
-          {
-            "content": {
-              "ssvc_v1": {
-                "id": "CVE-1900-0001",
-                "schemaVersion": "1-0-1",
-                "selections": [
-                  {
-                    "name": "Technical Impact",
-                    "namespace": "x_custom",
-                    "values": [
-                      "Total"
-                    ],
-                    "version": "1.0.0"
-                  }
-                ],
-                "timestamp": "2024-01-24T10:00:00.000Z"
-              }
-            },
-            // ...
-          }
-        ]
-      }
-    ]
-  }
-```
-
-> The namespace `x_custom` is a private namespace.
-> Its decision point definitions might therefore not be known to the reader of the document.
-
-### Usage of SSVC Decision Point Namespace with Extension in non TLP:CLEAR Document
-
-For each SSVC decision point given under `selections`, it MUST be tested the `namespace` does not use an extension
-if the document is not labeled `TLP:CLEAR`.
-
-The relevant path for this test is:
-
-```
-   /vulnerabilities[]/metrics[]/content/ssvc_v1/selections[]/namespace
-```
-
-*Example 1 (which fails the test):*
-
-```
-  {
-    "document": {
-      // ...
-      "distribution": {
-        "tlp": {
-          "label": "GREEN"
-        }
-      },
-      // ...
-    }
-    "vulnerabilities": [
-      {
-        "cve": "CVE-1900-0001",
-        "metrics": [
-          {
-            "content": {
-              "ssvc_v1": {
-                "id": "CVE-1900-0001",
-                "schemaVersion": "1-0-1",
-                "selections": [
-                  {
-                    "name": "Technical Impact",
-                    "namespace": "ssvc/additional-technical-impacts",
-                    "values": [
-                      "Total"
-                    ],
-                    "version": "1.0.0"
-                  }
-                ],
-                "timestamp": "2024-01-24T10:00:00.000Z"
-              }
-            },
-            // ...
-          }
-        ]
-      }
-    ]
-  }
-```
-
-> The namespace contains the extension `additional-technical-impacts`.
-> Its decision point definitions might therefore not be known to the reader of the document.
-
```

Comment:
@sthagen: Please restore these - it is not a copy-paste mistake: They look like the optional tests but  in fact they are different. The optional tests cover the `TLP:CLEAR` case. These ones here cover the non `TLP:CLEAR` cases.

Reasoning: In a closed user group, the usage of private extensions and private decision points have a valid use case.

Reply:
Ah, sorry. Looked very similar (I did on spot a difference) and then the convenient theory sneaked in of a half-completed move. Problem is, that the headings cannot be left as is (the link part). They need disambiguation from the others. Otherwise, the rendering / display device does not know where to jump to.

Reply:
Done in [b3e4578](https://github.com/oasis-tcs/csaf/pull/929/commits/b3e45783f0a1f67575f1ffb05850ae97a1f87b1c):

```
commit b3e45783f0a1f67575f1ffb05850ae97a1f87b1c (HEAD -> editor-revision-2025-03-26, origin/editor-revision-2025-03-26)
Author: Stefan Hagen <stefan@hagen.link>
Date:   Fri May 2 01:14:02 2025 +0200

    Reverted the removal of two informative test sections

    - fixed an oversight where long section titles with an added term non
      were taken to be duplicates
    - thanks to Thomas Schmidt for noticing.
    - added missing section and example entries to mappings

    Signed-off-by: Stefan Hagen <stefan@hagen.link>
```